model/storage/ConsumerStorage.php

```diff
@@ -73,21 +77,52 @@ public function setConsumerToken(\core_kernel_classes_Resource $consumer, Access
      *
      * Fetch token from cache if exists, otherwise set it
      *
-     * @param $tokenHash
+     * @param string $tokenHash
      * @return AccessToken
-     * @throws \common_Exception
-     * @throws \common_exception_NotFound
-     * @throws \core_kernel_persistence_Exception
+     * @throws common_Exception
+     * @throws common_exception_NotFound
```

Comment:
No need to add throw `NotFound` phpdoc, as we throw it inside this method

Reply:
The PHP doc is for the clients of this class, not for the developer of the class. So in theory we should add all the possible exception that this class throws.

Reply:
We decided to make it optional, so I am keeping them :)